composer.json

```diff
@@ -2,12 +2,13 @@
     "name": "stfc-egi/gocdb",
     "description": "A web portal and REST style API for e-Infrastructure topology managment",
     "require": {
-	"doctrine/orm": "2.4.8",
-	"doctrine/dbal": "2.5.4"
-    }, 
+	"doctrine/orm": "2.11.*",
```

Comment:
This wild card is probably fine, as the doctrine ORM will be held to a specific version by composer.lock unless that file is regenrated manually.

Rolling this out will require some thought from @gregcorbett.